### ISSUE TITLE
[Bug] Rule 0511 'empty-metadata' does not work

### ISSUE THREAD

**Issue by MaximeAbout** [MarketSquare/robotframework-robocop#1040]
### What happened?

Create a simple TS without a Metadata info in test suite settings:

```
*** Settings ***

*** Test Cases ***
TEST
    [Tags]  xpu32  esxi  conftool
    No Operation

TEST2
    No Operation

TEST3
    [Tags]  xpu32
    No Operation

TEST4
    [Tags]  toto
    No Operation
```

### What command/code did you try to run?

robocop -i 0511 bad_ts.robot

### What is the full error message?

Nothing

### What did you expect to happen instead?

bad_ts.robot:37:1 [W] 0511 Metadata settings does not have any value set

### Operating System

Linux

### Robocop version

4.2.1

**Comment by bhirsz** [comment]
It does work - but not in that way. Various ``empty-X`` rules are checking if given rule exist but without any value:

```
*** Settings ***
Metadata  # will report
```

Missing setting does not raise anything. If you need rule that does check if given setting exist you would need to create your own custom rule or it could be added to our repository as community rule. As for custom rule it should be as easy as:


```
rules = {
    "9999": Rule(
        rule_id="9999",
        name="missing-metadata",
        msg="There is no metadata set",
        severity=RuleSeverity.WARNING,
    ),
}
class MissingSettingsChecker(VisitorChecker):
    reports = ("missing-metadata",)

    def __init__(self):
        self.metadata_present = False
        super().__init__()

    def visit_File(self, node):
        self.metadata_present = False
        self.generic_visit(node)
       if not self.metadata_present:
            self.report("missing-metadata", node=node)

    def visit_Metadata(self, node):  # noqa
        self.metadata_present = True
  

```

**Comment by MaximeAbout** [comment]
Got it, thank you !
